samples/bluetooth/peripheral_hids_keyboard/README.rst

```diff
@@ -98,8 +98,9 @@ Testing with a Microsoft Windows computer
 To test with a Microsoft Windows computer that has a Bluetooth radio, complete the following steps:
 
 1. Power on your development kit.
-#. Press Button 4 on the kit if the device is not advertising.
+#. Press **Button 4** on the kit if the device is not advertising.
    Advertising is indicated by blinking **LED 1**.
+#. Make a UART connection between the DUT and your computer.
```

Comment:
Probably the following may be copied from peripheral_uart's README: "Connect to the kit with a terminal emulator (for example, PuTTY). See How to connect with PuTTY for the required settings."